### DISCUSSION TITLE
sauceNAO上不去，导致搜图功能一直只能用ascii2d

### DISCUSSION THREAD

**Discussion by luenhua** [Tsuk1ko/CQ-picfinder-robot#307] [问题求助] [answered]

**Comment by Tsuk1ko** [comment, marked as answer]
配置 `bot.proxy`

https://github.com/Tsuk1ko/cq-picsearcher-bot/blob/20deb3403ccbaf6963d011ec347df2ebc68f8c08/config.default.jsonc#L68-L69

**Reply by luenhua** [reply]
谢谢大佬
